data/randomization/randomization-with-weather.ipynb

```diff
@@ -0,0 +1,732 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.                selected_rows = selected_rows.append(selected_extreme_row).append(selected_normal_row)
</pre><p>I think it would be good for <code>check_extreme_conditions</code> to return the type of extreme condition(s) the given row contains, since it is heavy rain OR high wind, and put this information into the <code>selected_rows</code> dataframe so that it is easy to see under what condition a specific file is selected (normal or heavy rain and/or high wind)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/bat-detector-validation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>